[Platform][Store] Always batch embeddings, drop catalog coupling (#2230)
symfony/ai: src/platform/CHANGELOG.md

0.11
----

>  * [BC BREAK] Remove `Capability::INPUT_MULTIPLE` from all embedding models in the model catalogs (kept on reranking models); every embedding bridge already accepts multiple inputs in a single API call, so the capability no longer carried meaning for embeddings

Author: ```suggestion
 * [BC BREAK] Remove `Capability::INPUT_MULTIPLE` from all embedding models since every embedding bridge already accepts multiple inputs in a single API call
```
